machine_management/index-test.adoc

```diff
@@ -0,0 +1,43 @@
+include::_attributes/common-attributes.adoc[]
+:context: index-test-gh-comments
+
+toc::[]
+
+Here is a file to mess with to try to accept comments directly via GH review features.
+Text from the link:https://www.lipsum.com[Lorem Ipsum] generator.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean consectetur placerat purus vel pulvinar. Ut id dignissim erat. Nam volutpat consequat posuere. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce massa dolor, interdum egestas imperdiet non, accumsan id lorem. Sed sed ultrices quam, nec aliquet orci. Cras sodales ac quam id hendrerit. Vivamus et eleifend ipsum, sed iaculis lorem. Nulla scelerisque, ex a dapibus dignissim, ipsum libero hendrerit neque, ac scelerisque odio sapien quis ipsum. Maecenas in hendrerit risus. Nullam eget sapien massa. Donec magna quam, vehicula eu elementum at, volutpat et est. Cras vel suscipit nisi, aliquam congue mauris. In fermentum nulla sed tortor imperdiet tristique.
+
+[NOTE]
+====
+Nam nec dolor ut orci ultrices volutpat eu sit amet urna. In diam quam, aliquam ut tristique id, laoreet id tortor. Aliquam convallis ac lorem varius elementum. Integer auctor velit pretium, molestie turpis et, ornare arcu. Nam non orci at enim fringilla congue eget nec tortor. Duis tempor finibus felis quis bibendum. Vestibulum a metus aliquam, sollicitudin felis id, sagittis ante.
+====
+
+In bibendum tristique mauris, in tincidunt lectus sagittis et. Fusce egestas vehicula purus eget fermentum. Donec elit risus, dapibus non vulputate sed, ultrices vitae sapien. Sed ac sapien at sapien luctus efficitur ac sit amet sapien. Nulla imperdiet tempus aliquet. Pellentesque finibus, lectus sed suscipit vestibulum, nisl tellus laoreet elit, eget scelerisque mi lacus at mi. Nullam ut arcu leo.
+
+Integer lacinia sagittis quam eu euismod. Nullam aliquam, odio hendrerit bibendum sodales, tortor tortor ultrices ex, at dictum libero sem eget tortor. Nulla facilisi. Nunc vel laoreet ipsum. Ut at enim in lorem commodo semper eget nec mi. Nullam in malesuada sapien. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Pellentesque pretium sit amet nisi in varius. Suspendisse potenti. Duis vel lacus sem. Sed pulvinar, leo quis sollicitudin ullamcorper, dui lacus gravida enim, at vestibulum lorem nisi at diam. Praesent faucibus porttitor nisi, at fringilla libero sagittis quis. Etiam eleifend erat quis auctor rhoncus. Quisque porttitor nunc in mollis commodo.
+
+. Fusce a vulputate nisl, vel porttitor mauris:
+** Cras cursus euismod libero eu maximus. Sed ac efficitur felis.
+** Nullam posuere nec sapien sit amet maximus.
+Cras venenatis euismod pharetra.
+. Nunc ut facilisis purus, ac convallis leo.
+Pellentesque non metus ac elit mattis consequat in sit amet sapien.
+. Etiam maximus tellus a libero aliquam, vel eleifend dui ultrices.
++
+[source,terminal]
+----
+Nunc dapibus facilisis elit, quis sodales tortor rutrum dapibus.
```

Comment:
```suggestion
Nunc dapibus facilisis elit, quis sodales <tortor_rutrum> dapibus. # <1>
```
<1> carpe diem!

Reply:
Interesting, because I put <1> carpe diem! in the edit, but it tossed it down out of the field.

Reply:
That is a bit weird!